src/python/pcmdi/scripts/pcmdi_metrics_driver.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/home/igcmg/PCMDI-MP/devel_install_PCMDI_IPSL/PCMDI_METRICS/bin/python
```

Comment:
@doutriaux1 I don't even know why this line is here...
After looking at the driver in the branch, I really think I've done something wrong. I don't even see the modifications to compute only the custom metrics... I'm not a git master yet...
I'll check what I've done and get back to you with another PR.